views/admin/rooms.ejs

```diff
@@ -14,11 +14,11 @@
 		<% for(var i=0; i<rooms.length; i++) {%>
 		<tr>
 			<td><%= rooms[i].room %></td>
-		  	<td><a href="<%= __base_url + "/room/" + rooms[i].room %>"><%= rooms[i].title %></a></td>
+		  	<td><a href="/<%= "/room/" + rooms[i].room %>"><%= rooms[i].title %></a></td>
```

Comment:
we can probably just do `/room/<%= rooms[i].room%>` instead of the extra string append